pkg/controller/installation/core_controller.go

```diff
@@ -503,6 +512,18 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, err
 	}
 
+	var aci *operator.AmazonCloudIntegration
+	if instance.Spec.Variant == operator.TigeraSecureEnterprise {
+		aci, err = utils.GetAmazonCloudIntegration(ctx, r.client)
```

Comment:
Making a note that we'll need RBAC changes for this (haven't check to see if they exist in this PR yet)

Reply:
I don't think an RBAC change is needed because the operator has access to all `operator.tigera.io` apiGroup resources.